tests/update/updates_packagekit_kde.pm

```diff
@@ -45,20 +45,27 @@ sub run {
 
             # Wait until installation is done
             assert_screen \@updates_installed_tags, 3600;
-
             # Make sure the applet has fetched the current status from the backend
             # and has finished redrawing. In case the update status changed after
             # the assert_screen, record a soft failure
             wait_still_screen;
+
+            #Check if the applet reports that a restart is needed.
+            if (check_screen 'updates_installed-restart') {
+                assert_and_click 'updates_installed-restart', 5;
+                $self->wait_boot;
+                setup_system;
+                # Look again
+                assert_screen \@updates_installed_tags;
```

Comment:
This enters the desktop again, but not in the "updates just installed" state, but the "updates just checked" one.
So technically it should repeat the check starting from line 35 and not here. Is there a difference between `updates_none` and `tray-without-updates-available`?